Doc/library/os.path.rst

```diff
@@ -65,7 +65,8 @@ the :mod:`glob` module.)
 
    Return a normalized absolutized version of the pathname *path*. On most
    platforms, this is equivalent to calling the function :func:`normpath` as
-   follows: ``normpath(join(os.getcwd(), path))``.
+   follows: ``normpath(join(os.getcwd(), path))``.  Unlike :func:`realpath`,
+   this function preserves symlink names.
```

Comment:
I think this should be phrased in the negative that "this function does not resolve symlink targets". 

If the reader follows through to the documentation of `normpath`, it's clear that it resolves ".." components, which may remove symlinks from the path. But based on just this sentence, it's not clear what it means to preserve symlinks. A different implementation might retain all ".." components, as pathlib's `absolute` method does. But the current implementation can change the real target of a Unix path, e.g. `normpath('spam/symlink/../eggs')` -> "spam/eggs".

In Windows, this is usually not a problem since the Windows API works like this anyway. The one exception is in the target of a relative symlink, e.g. `symlink(r'spam\symlink\..\eggs', 'eggs')` is not the same as `symlink(r'spam\eggs', 'eggs')`.

Reply:
I updated the wording to more closely reflect, in a negative fashion, the language used for realpath().